build/lib/singa_auto/advisor/constants.py

```diff
@@ -20,7 +20,7 @@
 from enum import Enum
 from typing import Union
```

Comment:
Do you copy files from singa_auto to build/lib/singa_auto? 
Can copy them when creating the pip package instead of putting them into the repo?

Reply:
yes, we can copy some file based on our needs